packages/drops/src/queries/SearchDrops.ts

```diff
@@ -0,0 +1,60 @@
+import { DropResponse } from '../types/DropResponse';
+
+export const SEARCH_DROPS_QUERY = `
+  query SearchDrops(
+    $limit: Int!
+    $offset: Int!
+    $args: search_drops_args!
+    $orderBy: [drops_order_by!]
+  ) {
+    search_drops(
+      limit: $limit
+      offset: $offset
+      args: $args
+      order_by: $orderBy
+    ) {
+      id
+      fancy_id
+      name
+      description
+      city
+      country
+      channel
+      platform
+      location_type
+      drop_url
+      image_url
+      animation_url
+      year
+      start_date
+      timezone
+      private
+      created_date
+      expiry_date
+      end_date
+      stats_by_chain_aggregate {
+        aggregate {
+          sum {
+            transfer_count
+            poap_count
+          }
+        }
+      }
+      email_claims_stats {
+        total
+      }
+      drop_image {
```

Comment:
The search function is exclusively used to display a list of collections, the results. In that sense, the original drop image should not be needed. I'm ok with it being there, as it's probably the expectation, but wanted to comment as it's a join statement that is executed while not needed.